operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -87,12 +87,26 @@ default int concurrentReconciliationThreads() {
     return DEFAULT_RECONCILIATION_THREADS_NUMBER;
   }
 
+  default int minConcurrentReconciliationThreads() {
+    return MIN_DEFAULT_RECONCILIATION_THREADS_NUMBER;
+  }
+
   int DEFAULT_WORKFLOW_EXECUTOR_THREAD_NUMBER = DEFAULT_RECONCILIATION_THREADS_NUMBER;
+  int MIN_DEFAULT_WORKFLOW_EXECUTOR_THREAD_NUMBER = MIN_DEFAULT_RECONCILIATION_THREADS_NUMBER;
 
+  /**
+   * Retrieves the maximum number of threads the operator can spin out to be used in the workflows.
+   *
+   * @return the maximum number of concurrent workflow threads
+   */
   default int concurrentWorkflowExecutorThreads() {
     return DEFAULT_WORKFLOW_EXECUTOR_THREAD_NUMBER;
   }
 
+  default int minConcurrentWorkflowExecutorThreads() {
```

Comment:
Would be nice to add doc for this as well.

Reply:
added docs